packages/runtime/runtime-definitions/src/garbageCollection.ts

```diff
@@ -3,19 +3,24 @@
  * Licensed under the MIT License.
  */
 
+// The key to use for storing garbage collection blob in summary.
+export const gcBlobKey = "gc";
+
 /**
  * Represents the garbage collection data returned by each node in the Container. It is used for
  * running GC in a document.
  */
-export interface IGCData {
+export interface IGarbageCollectionData {
     /** The GC nodes of a Fluid object in the Container. Each node has an id and a set of routes to other GC nodes. */
     gcNodes: { [ id: string ]: string[] };
 }
 
 /**
  * Represents the format of the GC details that is stored in the summary for each node.
  */
-export interface IGCDetails {
+export interface IGarbageCollectionDetails {
```

Comment:
Since this is being ranamed already.. I feel like it's hard for me to remember the difference between `IGarbageCollectionData` and `IGarbageCollectionDetails`, mainly because the terms "data" and "details" do not really provide a good distinction between what they are or how they are used.

I feel like `IGarbageCollectionDetails` should be renamed to indicate that it is the data within the **summary** blob.

Reply:
Good point! They do look very similar, naming wise. How about IGCSummaryDetails or IGarbageCollectionSummaryDetails?

Reply:
Sounds good to me.